tests/test_autotuner.py

```diff
@@ -140,10 +139,10 @@ def test_mode_destructive():
     op = Operator(Eq(f, f + 1.), dle=('advanced', {'openmp': False}))
     op.apply(time=100, autotune=('basic', 'destructive'))
 
-    # AT is expected to have executed 30 timesteps (6 block shapes, 5 timesteps each)
+    # AT is expected to have executed 18 timesteps (6 block shapes, 3 timesteps each)
     # The operator runs for 101 timesteps
-    # So, overall, f.data[0] is incremented 131 times
-    assert np.all(f.data == 131)
+    # So, overall, f.data[0] is incremented 119 times
+    assert np.all(f.data == 119)
```

Comment:
So what's the reason behind reducing the number of shapes tried?

Reply:
it's not the number of shapes tried, this is just due to the number of timesteps per shape . Ie, due to the change to ``at_squeezer`` above

Reply:
I mean since when AT is modifying the ouput? No matter how many block you try you still run `f=f+` 100 times no matter what AT does so should always be `f.data=1` this is super dangerous if not the case.

Reply:
ah, are you not aware that we have had for quite sometime "runtime autotuning" ? this is simply performing autotuning at runtime for some blockshapes

Reply:
Yes I know, but runtime AT means it  uses the first 13 steps to do AT then does the rest, but the overall should still be 100 steps, not 113.

Reply:
sorry, this isn't 'runtime' mode, this is 'destructive', which essentially doesn't allocate new data, hence the 119